src/addons/withSoundCloudAudio.js

```diff
@@ -142,9 +169,18 @@ export default function withSoundCloudAudio (WrappedComponent) {
     }
 
     onCanPlay() {
-      const { onCanPlayTrack } = this.props;
+      const { onCanPlayTrack, startTime } = this.props;
 
-      onCanPlayTrack && onCanPlayTrack(this.soundCloudAudio);
+      if (startTime && !this.state.hasSetStartTime) {
+        this.soundCloudAudio.setTime(startTime);
+
+        this.setState({
+          hasSetStartTime: true,
+          currentTime: this.soundCloudAudio.audio.currentTime
+        });
+      } else {
```

Comment:
I added this else statement, since setting the time causes onCanPlay to fire a few extra times.